src/sif/autocost.cc

```diff
@@ -45,6 +45,7 @@ constexpr float kDefaultUseFerry = 0.5f;     // Factor between 0 and 1
 constexpr float kDefaultUseRailFerry = 0.4f; // Factor between 0 and 1
 constexpr float kDefaultUseHighways = 1.0f;  // Factor between 0 and 1
 constexpr float kDefaultUseTolls = 0.5f;     // Factor between 0 and 1
+constexpr float kDefaultUseDistance = 0.f;   // Factor between 0 and 1
```

Comment:
the default is to care only about minimizing time (as it always was) which means there are zero RAD differences

Reply:
Could the comment or variable name itself mention that it weights between minimizing time or distance?

Feel free to ignore if it doesn't make sense.

Reply:
sure!